hackathon_site/dashboard/frontend/src/components/inventory/Item/Item.module.scss

```diff
@@ -0,0 +1,46 @@
+@import "./../../../assets/abstracts/mixins";
+@import "./../../../assets/abstracts/variables";
+
+.Item {
+    display: flex;
+    width: 100%;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: flex-start;
+    padding-bottom: 10px;
+
```

Comment:
Forgot to mention last time sorry:

Can you please put `cursor: pointer;` under `.Item` so it's the little hand cursor when you hover over it and the user knows its clickable